### PR TITLE
Home static component with new UI for navigation design

### PR DESCRIPTION
**Pull request by stefancrowdbotics**
Screenshot 2020-07-02 at 01 46 49

### CODE REVIEW

**Comment by stefancrowdbotics** [comment]
Screenshot 2020-07-02 at 13 37 29

File: src/styles/components/HomeStatic.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
+.container {
+  display: flex;
+  width: 100%;
+  height: 548px;
+  background-image: url('../../assets/images/homeStaticSlider.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```
Comment (kriskross2893) [inline src/styles/components/HomeStatic.css:10]: on my screen it does not cover the whole width of the page.

Selection_091

if you add `background-size: cover` here it should fix the problem

File: src/styles/components/HomeStatic.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+.infoDividerContainer {
+  display: flex;
+  height: 440px;
+  width: 100%;
+  background-image: url('../../assets/images/homeStaticSlider.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```
Comment (kriskross2893) [inline src/styles/components/HomeStatic.css:134]: this one also does not cover whole width.

Selection_092

add `background-size:cover` and it should be fine

File: src/styles/components/HomeStatic.css [hunk 3]
```diff
@@ -3,7 +3,7 @@
   width: 100%;
   height: 548px;
   background-image: url('../../assets/images/homeStaticSlider.png');
-  background-position: center;
+  background-position: cover;
```
Comment (kriskross2893) [inline src/styles/components/HomeStatic.css:6]:
```suggestion
  background-size: cover;
```

Comment (kriskross2893) [reply src/styles/components/HomeStatic.css:6]: you mistype size with position

File: src/styles/components/HomeStatic.css [hunk 4]
```diff
@@ -127,7 +127,7 @@
   height: 440px;
   width: 100%;
   background-image: url('../../assets/images/homeStaticSlider.png');
-  background-position: center;
+  background-position: cover;
```
Comment (kriskross2893) [inline src/styles/components/HomeStatic.css:130]: this one also, you mistype background-size with background-position
```suggestion
  background-size: cover;
```

Comment (stefancrowdbotics) [reply src/styles/components/HomeStatic.css]: You are right, missed prop in css for cover

**Pull request merged**